staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -114,14 +130,23 @@ func (c *crConverter) ConvertFieldLabel(gvk schema.GroupVersionKind, label, valu
 }
 
 func (c *crConverter) Convert(in, out, context interface{}) error {
+	// Special-case typed scale conversion if this custom resource supports a scale endpoint
```

Comment:
The integration test doesn't cover the "CR/scale" case. Can we add a test?

Reply:
actually, it does... that's how I found this case was broken